.babelrc

```diff
@@ -1,3 +1,3 @@
 {
-  "presets": ["es2015", "react"]
+  "presets": ["es2015", "react", "stage-0"]
```

Comment:
Destructuring is provided by es2015 isn't it?  stage-0 provides `bind` syntax and a `do` statement.